codacy/templates/rabbitmq-secret.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```

Comment:
This ensures that the rabbitmq queues are recreated when the statefulset is restarted